src/test/java/com/netcracker/study/general/GeneralTest.java

```diff
@@ -0,0 +1,14 @@
+package com.netcracker.study.general;
+
+import com.netcracker.study.objects.borders.*;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class GeneralTest {
```

Comment:
rename test to BordersFieldTest. Tests should be named as {ClassName}Test to easily naviagate among them